### DISCUSSION TITLE
SFM3300 Arduino

### DISCUSSION THREAD

**Discussion by amamethven** [Nabilphysics/ventilator#3] [General]
Hi there,
I am hoping to put together a simple device that we can use to obtain ventilator outputs such as flow, pressure and leak. I am NOT an engineer and would like to use a basic Arduino setup, if possible, to pull data from the sensirion flow/pressure meter with one of your cables and output the data as an analog signal to our data logger/peripheral device input.
![image](https://user-images.githubusercontent.com/96119886/162651203-7092b2b0-794d-4c65-ad8c-a72a43247734.png)

  (Sorry about the picture, as I said I am not an engineer!)
From your work do you know which coloured wires are SCL and SDA? I am presuming Brown is +5V and Blue is GND as this is what they have used in other evaluation cables. If you don't know which colour is SDA or SCL do you know a way to find out?
![image](https://user-images.githubusercontent.com/96119886/162651031-d0a45604-b11e-468d-a509-a445cc50e3a3.png)
Thanks & I love your project!

**Comment by Nabilphysics** [comment]
Dear,
To be able to determine which wire is which first you have to see the
datasheet of the sensor. Then from datasheet you have to identity the
connectors pin first. Then plz use a multimeter in continuity mode to be
able to identify corresponding colour of that perticular pin.
